### DISCUSSION TITLE
[Help] Is this repo dead ?

### DISCUSSION THREAD

**Discussion by zcyc** [fomantic/Fomantic-UI#1461] [General] [answered]
# Help Wanted
no update for a long time. i like this ui framework.

**Comment by dutrieux** [comment]
You can check the dev branch : https://github.com/fomantic/Fomantic-UI/commits/develop and différents pulls requests.
I think is still very active :)

**Comment by lubber-de** [comment, marked as answer]
See https://github.com/fomantic/Fomantic-UI/tree/develop
Last change: 7 days ago...which i would not call "a long time" 😄 

As @dutrieux pointed out, all Pull requests are merged into the `develop` branch. The `master`  branch is only kept for official releases, as we are collecting pull requests until a reasonable amount for another official release.
Thus, the master branch will only change every few weeks.

If you would like to always get the latest version, try the nightly build out of the develop branch which you can find via npm/jsdelivr

https://cdn.jsdelivr.net/npm/fomantic-ui@nightly/dist/semantic.min.js
https://cdn.jsdelivr.net/npm/fomantic-ui@nightly/dist/semantic.min.css

Of course the activity will increase if more people from the community would be able to help and support the project by providing proper PRs which will fix open bugs or new features 😉 

And:
Compare the activity to the original Semantic UI repo, which hasn't got any commits since 2 years :
 😉

**Comment by zcyc** [comment]
very thank you
